SConstruct

```diff
@@ -232,6 +232,20 @@ else:
     if selected_platform != "":
         print("Automatically detected platform: " + selected_platform)
 
+if selected_platform in ["osx", "macos"]:
+    if selected_platform == "macos":
+        # Alias for forward compatibility.
+        print('Platform "macos" is still called "osx" in Godot 3.x. Building for platform "osx".')
+    # Alias for convenience.
+    selected_platform = "osx"
+
+if selected_platform in ["iphone", "ios"]:
+    if selected_platform == "ios":
+        # Alias for forward compatibility.
+        print('Platform "ios" is still called "iphone" in Godot 3.x. Building for platform "iphone".')
+    # Alias for convenience.
+    selected_platform = "iphone"
```

Comment:
You don't need to check for `osx` and `iphone`, only for the new names.

Reply:
Fixed